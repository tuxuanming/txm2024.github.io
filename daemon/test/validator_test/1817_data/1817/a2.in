10
20
30
40
50
49
48
47
