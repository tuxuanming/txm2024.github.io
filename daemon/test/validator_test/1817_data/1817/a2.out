507
5622109
95991365288
1954002050661819
4262465330
16033495247557039074
5873225808361331954
2152814945971655556
